r-package/grf/R/average_late.R

```diff
@@ -62,7 +62,7 @@ average_late <- function(forest,
           "only implemented for binary instruments."
       ))
   }
-  
```

Comment:
All team grf @davidahirshberg @swager @halflearned @rinafriedberg : Can you make sure your text editor removes white space when you save? That will keep this from popping up in diffs.